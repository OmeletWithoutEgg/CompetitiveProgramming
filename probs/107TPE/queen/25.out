293175.49 -52558038.45
