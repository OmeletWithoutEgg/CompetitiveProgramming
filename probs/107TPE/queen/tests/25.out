-11052007.63 57036072.58
